src/EditorFeatures/CSharpTest/NavigateTo/NavigateToTests.cs

```diff
@@ -1473,6 +1474,36 @@ public class C
                 },
                 await _aggregator.GetItemsAsync("C"));
         }
+
+        [Fact]
+        public async Task DoIncludeSymbolsFromMultipleSourceGeneratedFiles()
+        {
+            using var workspace = TestWorkspace.CreateCSharp(
```

Comment:
can the xml syntax cannot support this scenario?

Reply:
all the other tests in this file use the xml syntax so I would prefer if we kept things consistent

Reply:
➡️ I needed one test to use the new code so we can see it's working. Both work; but the XML form is clumsier. Eventually it would be nice to provide an object model similar to Microsoft.CodeAnalysis.Testing.

Reply:
> but the XML form is clumsier

Is this a style question or are there bugs that we don't find by testing with one or the other? 

I can agree that testing via XML is much nicer in VB and would be fine if we used a different teat api surface area in C# vs VB. However, without some additional benefit beyond ergonomics I can't say I think it would be worth the exercise.

Reply:
It doesn't have a schema, so there's no real completion support. I'm not a fan of the reliance on copy/pasting from some other existing test which occurs even when writing VB code.